languages/elixir/exercises/concept/binary-matching/.docs/introduction.md

```diff
@@ -0,0 +1,53 @@
+Elixir provides an elegant syntax working with binary data as we have seen with the `<<>>` special form provided for working with _bitstrings_.
+
+## Binary data
+
+The binary type is a specialization on the bitstring type. Where bitstrings could be of any length (any number of [bits][wiki-bit]), binaries are where the number of [bits][wiki-bit] can be evenly divided by 8. That is, when working with binaries, we often think of things in terms of [bytes][wiki-bytes] (8 [bits][wiki-bit]). A [byte][wiki-bytes] can represent integer numbers from `0` to `255`. It is common to work with [byte][wiki-byte] values in [hexadecimal][wiki-hexadecimal], `0x00 - 0xFF`.
+
+Binaries literals are defined using the bitstring special form `<<>>` to define a literal. When defining a literal, we can use integer and string literals. Integer values greater than 255 will overflow and only the last 8 bits of the integer will be used. By default, the `::binary` modifier is applied to the value. We can concatenate binaries with the `<>/2` operator.
+
+```elixir
+<<255>> == <<0xFF>>
+<<256>> == <<0>> # Overflowing bits are truncated
+<<"Hello">> == <<72, 101, 108, 108, 111>>
+```
+
+A _null-byte_ is another name for `<<0>>`
+
+## Strings are encoded binary data
```

Comment:
Now that I look at the whole PR, I am not sure if this paragraph should be in the introduction. It's not needed to solve the exercise, right?

Reply:
The way github displays your comment, I'm not sure which paragraph you are referring to.

Reply:
But isn't it enough that this is mentioned in `after.md`?

Reply:
> But isn't it enough that this is mentioned in `after.md`?

I think so. This sounds a bit like it is useful information that is directly related to the Concepts, but not necessary to know up-front (as in, in the `introduction.md` document). It is perfectly reasonable to have something like this only be mentioned in the `after.md` document, unless there is enough to warrant an entire new exercise of course.